ubuntu/debian/compat

```diff
@@ -1 +1 @@
-9
+11
```

Comment:
checking to see if this is intended? Looking at a few other release repos, the debhelper compatibility levels are set to 9.

Reply:
I made the change after seeing https://github.com/gazebo-release/sdformat13-release/pull/18.

Reply:
ah ok I see

Reply:
maybe we also need to change this from 13 to 11: https://github.com/gazebo-release/sdformat14-release/blob/ci_matching_branch/bump_harmonic_sdformat14/jammy/debian/compat

Reply:
Oh interesting, I think we actually need 13, so I'll update this file and make the jammy one symlink to this. I found https://manpages.ubuntu.com/manpages/jammy/en/man7/debhelper.7.html#compatibility%20levels which states 13 is the recommended level for Jammy. For Focal, https://manpages.ubuntu.com/manpages/focal/en/man7/debhelper.7.html#compatibility%20levels says 12 is recommended and actually says not to use 11.

Reply:
sounds good!